app/models/menu.ts

```diff
@@ -242,16 +242,11 @@ if (process.platform === 'darwin') {
     ]);
 }
 
-/**
- * Only allow the Developer Tools to be triggered in a development build.
- */
-if (process.env.ENV === 'development') {
-    let viewSubmenu = viewMenu.submenu;
-    if (viewSubmenu && viewSubmenu instanceof Array) {
-        viewSubmenu.push({
-            role: 'toggledevtools',
-        });
-    }
+let viewSubmenu = viewMenu.submenu;
```

Comment:
`(node:75895) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOENT: no such file or directory, open '/Users/Danie
l/Development/sinap-ide/build/package.json'`

I'm gonna look into this but any ideas @slaymaker1907 ?